### PR TITLE
Feature/service status update to hmi

### PR DESCRIPTION
**Pull request by mked-luxoft**
Implements [#2791](https://github.com/smartdevicelink/sdl_core/issues/2791)

This PR is **ready** for review.

### Testing Plan
Manual testing only.

### Summary
Add implementation for Service Status Update to HMI feature

### CLA
- [x] I have signed [the CLA](https://docs.google.com/forms/d/e/1FAIpQLSdsgJY33VByaX482zHzi-xUm49JNnmuJOyAM6uegPQ2LXYVfA/viewform)

### CODE REVIEW

**Comment by mked-luxoft** [comment]
@yang1070 This PR was updated to comply with proposal updates. Please review.

**Comment by AStasiuk** [comment]
@theresalech, this PR is already reviewed and approved by Ford (by @yang1070 ) and ready for Livio review.

**Comment by mked-luxoft** [comment]
This PR has been rebased on the latest develop branch and is ready for review

File: ffw/BasicCommunicationRPC.js [hunk 1]
```diff
@@ -135,6 +138,9 @@ FFW.BasicCommunication = FFW.RPCObserver
           .subscribeToNotification(this.onSDLConsentNeededNotification);
         this.onResumeAudioSourceSubscribeRequestID = this
           .subscribeToNotification(this.onResumeAudioSourceNotification);
+        this.onServiceUpdateNotificationSubscribeRequestID = this.client
+          .subscribeToNotification(this.onServiceUpdateNotification);
+        this
```
Comment (jacobkeeler) [inline ffw/BasicCommunicationRPC.js]: Incomplete statement

Comment (mked-luxoft) [reply ffw/BasicCommunicationRPC.js]: removed in ee2e38186fae19a88046d22be7ee9999b449f4eb

File: ffw/BasicCommunicationRPC.js [hunk 2]
```diff
@@ -297,11 +301,18 @@ FFW.BasicCommunication = FFW.RPCObserver
       onRPCNotification: function(notification) {
         Em.Logger.log('FFW.BasicCommunicationRPC.onRPCNotification');
         this._super();
+        if (notification.method == this.onServiceUpdateNotification) {
+          SDL.ServiceUpdatePopUp.activate(notification.params.serviceType,
```
[the remark this answers is not in the document]
Comment (ValeriiMalkov) [reply ffw/BasicCommunicationRPC.js]: @jacobkeeler Dispalying the popup for the RPC service type was removed in commit cacef97a38400c8a2d329103568dbb7b086decf1

File: ffw/BasicCommunicationRPC.js [hunk 3]
```diff
@@ -136,6 +136,8 @@ FFW.BasicCommunication = FFW.RPCObserver
           .subscribeToNotification(this.onSDLConsentNeededNotification);
         this.onResumeAudioSourceSubscribeRequestID = this
           .subscribeToNotification(this.onResumeAudioSourceNotification);
+        this.onServiceUpdateNotificationSubscribeRequestID = this.client
+          .subscribeToNotification(this.onServiceUpdateNotification);
```
[the remark this answers is not in the document]
Comment (ValeriiMalkov) [reply ffw/BasicCommunicationRPC.js]: @jacobkeeler Was rebased again and returned `this.onServiceUpdateNotification`

File: app/view/sdl/ServiceUpdatePopUp.js [hunk 4]
```diff
@@ -0,0 +1,237 @@
...
+        case 'INVALID_CERT': {
+          this.set('content', 'System is unable to authenticate the app');
+          break;
+        }
+        case 'INVALID_TIME': {
+          this.set('content', 'Unable to get valid time.' +
+            'Make sure your device has access to GPS signal');
+          break;
+        }
+        case 'PROTECTION_ENFORCED': {
```
[the remark this answers is not in the document]
Comment (mked-luxoft) [reply app/view/sdl/ServiceUpdatePopUp.js]: fixed in 03108c3

**Pull request merged**